### DISCUSSION TITLE
how to get UI elements inside Iphone AR environment

### DISCUSSION THREAD

**Discussion by streetcodec** [google/model-viewer#4386] [Q&A] [answered]
I want to show certain buttons and dropdowns while the person in viewing the model in AR. I did this by making the elements, children of the model viewer tag. This worked fine for Android and same doesn't work with iOS (Safari Browser). Is there a way to bring those elements, while AR Mode is active?

**Comment by danilokrulj** [comment, marked as answer]
Sadly no, there is no way to add any UI elements inside Quick-Looks AR mode. They are very strict about that. From what I know only thing you can add in AR is that iOS banner button or whatever it is, someone correct me if I'm wrong. That should be available once they accept WebXR mode on iOS, but who knows when will that happen.

**Reply by streetcodec** [reply]
Oh, that sucks. Thanks for your response though. I have another question regarding iOS thing, Can we somehow remove or customize the initial tutorial (Move the iphone around and that thing) and make it similar to Android implementation?

**Reply by danilokrulj** [reply]
I don't think so, maybe you can try and change the android one so they look the same or at least similar?

**Reply by streetcodec** [reply]
Hmm, Adding some tutorial thing at the beginning. Sounds plausible. Will try. Thanks!

**Reply by elalish** [reply]
@grorg - any hope for WebXR on iOS?
